File: app/src/main/java/org/tvheadend/tvhclient/ui/features/playback/internal/HtspDataSourceInterface.java
package org.tvheadend.tvhclient.ui.features.playback.internal;

public interface HtspDataSourceInterface {

    long getTimeshiftOffsetPts();

    void setSpeed(int tvhSpeed);

    long getTimeshiftStartTime();

    long getTimeshiftStartPts();

    void resume();

    void pause();
}

File: app/src/main/java/org/tvheadend/tvhclient/ui/features/playback/internal/HtspDataSourceInterface.kt
package org.tvheadend.tvhclient.ui.features.playback.internal

interface HtspDataSourceInterface {

    val timeshiftOffsetPts: Long

    val timeshiftStartTime: Long

    val timeshiftStartPts: Long

    fun setSpeed(tvhSpeed: Int)

    fun resume()

    fun pause()

    fun getResponseHeaders(): Map<String, List<String>>?
}
